lib/main/atom/autoCompleteProvider.ts

```diff
@@ -88,6 +89,53 @@ export class AutocompleteProvider implements Provider {
     return suggestions
   }
 
+  formatImportPath(sourcePath: string): string {
+    sourcePath = sourcePath.replace(/\.d$/, ""); // tsconfig.removeExt can convert d.ts file path into the filepath end with `.d`;
+    sourcePath = sourcePath.replace(/.*\/node_modules\//, "");
+    return sourcePath;
+  }
+
+  async getPathSuggestions(
+    prefix: string,
+    location: FileLocationQuery): Promise<SuggestionWithDetails[]> {
+    const client = await this.clientResolver.get(location.file)
+    const projectInfo = await client.executeProjectInfo({
+      file: location.file,
+      needFileNameList: true
+    })
+    const filePaths = projectInfo.body!.fileNames!.filter(p => p != location.file);
```

Comment:
The optional body for the `project-info` request is there for a reason - Typescript server might not always know what project the current file is in. It's up to you to handle it gracefully.